packages/react-textarea/Spec.md

```diff
@@ -2,62 +2,160 @@
+| [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/TextArea.html)  | `TextArea`                      |
+| [React Bootstrap](https://react-bootstrap.github.io/forms/input-group/)          | `FormControl` as `textarea`     |
+| [Blueprint](https://blueprintjs.com/docs/#core/components/text-inputs.text-area) | `TextArea`                      |
+| [Grommet](https://v2.grommet.io/textarea)                                        | `TextArea`                      |
+
+- Most libraries have their own `TextArea` component while others make it a variant of another component.
+  - `React Bootstrap` uses as with `FormControl` as follows: `<FormControl as="textarea" />`
+  - `Material UI` and `v8` use the `TextField` control and added a variant `multiline` as follows: `<TextField multiline defaultValue="Default Value" />`
+  - `Ant Design` uses the `TextArea` subcomponent as follows: `<Input.TextArea placeholder="Text Area" />`
+
+#### TextArea in v8
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+- `Textfield` component with `multiline` property. When the `multiline` property is set to true, it is rendered as a `textarea`.
+- Has descriptions, label, error message, and handles maximum character count.
+
+#### TextArea in v0
+
+- Doesn't have `label` prop.
+- Handles maximum character count.
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+```tsx
+<TextArea
+  className="textAreaClassName"
+  id="textarea-1"
+  defaultValue="This is a TextArea"
+  onChange={(ev, data) => console.log(data.value)}
+/>
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+- Visual variants
+  - `Filled Darker`: TextArea has a gray background and no outline.
+  - `Filled Lighter`: TextArea has a white background and no outline.
+  - `Underline`: TextArea has a gray background and a line on the bottom.
+  - `Outline`: TextArea has a white background, outline, and a line on the bottom.
+- Behavior variants
+  - `Disabled`: TextArea has a gray background, gray text, outline, and interaction is disabled.
+  - `Read Only`: TextArea has a gray background, gray outline, and typing is disabled.
 
 ## API
 
-_List the **Props** and **Slots** proposed for the component. Ideally this would just be a link to the component's `.types.ts` file_
+### Component props:
+
+```ts
+type TextAreaProps = {
+  /**
+   * Which direction the TextArea is allowed to be resized.
+   *
+   * @defaultvalue none
+   */
+  resize?: 'none' | 'horizontal' | 'vertical' | 'both';
+
+  /**
+   * Size of the TextArea.
+   *
+   * @defaultvalue medium
+   */
+  size?: 'small' | 'medium' | 'large';
+
+  /**
+   * Styling the TextArea should use.
+   *
+   * @defaultvalue outline
+   */
+  appearance?: 'outline' | 'underline' | 'filledDarker' | 'filledLighter';
```

Comment:
We should opt for kebab-case for consistency.

```suggestion
  appearance?: 'outline' | 'underline' | 'filled-darker' | 'filled-lighter';
```

Reply:
@andrefcdias It also not aligned in other components (`Input`, `Select`):

https://github.com/microsoft/fluentui/blob/3f3ef91846e63a345d9acba0e129656612d79b40/packages/react-select/src/components/Select/Select.types.ts#L36
https://github.com/microsoft/fluentui/blob/3f3ef91846e63a345d9acba0e129656612d79b40/packages/react-input/src/components/Input/Input.types.ts#L57

Would you mind to create an issue for this problem?